pvlib/pvsystem.py

```diff
@@ -1373,6 +1410,121 @@ def get_iam(self, aoi, iam_model='physical'):
             raise ValueError(model + ' is not a valid IAM model')
 
 
+@dataclass
+class AbstractMount(ABC):
+
+    @abstractmethod
+    def calculate_orientation(self, solar_zenith, solar_azimuth):
+        """
+        Determine module orientation.
+
+        Parameters
+        ----------
+        solar_zenith : numeric
+            Solar apparent zenith angle [degrees]
+        solar_azimuth : numeric
+            Solar azimuth angle [degrees]
+
+        Returns
+        -------
+        orientation : dict-like
+            A dict-like object with keys `'surface_tilt', 'surface_azimuth'`
+        """
+        pass
```

Comment:
```suggestion
        pass  # pragma: no cover
```

Reply:
In my experience you don't need a `pass` statement for an `@abstractmethod`. That would solve this without needing to exclude it from coverage.